client/src/pages/Profile/DriverForm.tsx

```diff
@@ -22,15 +23,15 @@ const ProfileForm: FC<IProfileForm> = ({
         <React.Fragment>
           <InputGeneral
             name="car_color"
-            validate={maxValue(15)}
+            validate={maxValueWithRequired(15)}
```

Comment:
create constants for each number for validation

Reply:
Resolved